pkg/observability/index.ts

```diff
@@ -131,7 +136,7 @@ export default function(plugin: IPlugin): void {
       name:       'observability',
       labelKey:   'observability.name',
       showHeader: false,
-      tooltip:    'Rancher Prime Observability through StackState',
+      tooltip:    'SUSE Rancher Prime Observability',
```

Comment:
does it make sense to use l10n for that text?

Reply:
It does not look like that defining `tooltip` adds a tooltip on hover, so I dropped that field.

Reply:
@aalves08 maybe you want to know about this ☝️ or maybe you know how to solve it (`tooltipKey` didn't work neither)